.github/workflows/run-all-tests.yml

```diff
@@ -79,10 +79,10 @@ jobs:
         shell: bash
 
       - name: Build
-        run: bazelisk build ${{env.BUILD_BUDDY_CONFIG}} --java_runtime_version=local_jdk_${{ matrix.jdk }} --disk_cache=${{ matrix.cache }} ${{ matrix.bazel_args }} //...
+        run: bazelisk build ${{env.BUILD_BUDDY_CONFIG}} --java_runtime_version=local_jdk_${{ matrix.jdk }} --disk_cache=${{ matrix.cache }} ${{ matrix.bazel_args }} --build_tag_filters="-no-${{ matrix.arch }}-jdk${{ matrix.jdk }},-no-jdk${{ matrix.jdk }}" //...
```

Comment:
Just want to verify that this is required: Do the tests that use this tag actually fail to build or just fail to run?

Reply:
True, it's not required for this test. In #643 I added (or better tried to) a test which only builds with later Java versions. So I thought, while I'm at it, I add `--build_tag_filters` as well. Adding a differently named set of build tags seems too much. WDYT?

Reply:
Agreed, if this is needed elsewhere than it's fine to add here.